diff --git a/dist/configs.js b/dist/configs.js
index 650f7a3b7ab8c13a831b77169a73a6ea770fdba0..162713d4d451233796731ae92284ec8e41c18df7 100644
--- a/dist/configs.js
+++ b/dist/configs.js
@@ -184,22 +184,7 @@ function customize(options = {}) {
       ignoreComments: false,
       ignoredNodes: [
         "TemplateLiteral *",
-        "JSXElement",
-        "JSXElement > *",
-        "JSXAttribute",
-        "JSXIdentifier",
-        "JSXNamespacedName",
-        "JSXMemberExpression",
-        "JSXSpreadAttribute",
-        "JSXExpressionContainer",
-        "JSXOpeningElement",
-        "JSXClosingElement",
         "JSXFragment",
-        "JSXOpeningFragment",
-        "JSXClosingFragment",
-        "JSXText",
-        "JSXEmptyExpression",
-        "JSXSpreadChild",
         "TSUnionType",
         "TSIntersectionType",
         "TSTypeParameterInstantiation",
@@ -296,7 +281,6 @@ function customize(options = {}) {
       "@stylistic/jsx-equals-spacing": "error",
       "@stylistic/jsx-first-prop-new-line": "error",
       "@stylistic/jsx-function-call-newline": ["error", "multiline"],
-      "@stylistic/jsx-indent": ["error", indent, { checkAttributes: true, indentLogicalExpressions: true }],
       "@stylistic/jsx-indent-props": ["error", indent],
       "@stylistic/jsx-max-props-per-line": ["error", { maximum: 1, when: "multiline" }],
       "@stylistic/jsx-one-expression-per-line": ["error", { allow: "single-child" }],
