diff --git a/dist/indent.js b/dist/indent.js
index 8a01873cf544b4f066e624f19b8e4e80a74327ed..dfd1146c3e7e14ae34ce246ceeb18f45dc3bc861 100644
--- a/dist/indent.js
+++ b/dist/indent.js
@@ -604,7 +604,7 @@ var indent = utils.createRule({
     }
     function validateTokenIndent(token, desiredIndent) {
       const indentation = tokenInfo.getTokenIndent(token);
-      return indentation === desiredIndent || indentation.includes(" ") && indentation.includes("	");
+      return indentation === desiredIndent;
     }
     function isOuterIIFE(node) {
       if (!node.parent || node.parent.type !== "CallExpression" || node.parent.callee !== node)
