diff --git a/PJV.js b/PJV.js
index 002ca2ae9021c3577fa81c7f7c42f820896b9b93..34363f3ab14ff2b4379f8743bd5c640d9ebd432f 100644
--- a/PJV.js
+++ b/PJV.js
@@ -12,13 +12,13 @@
 
     var PJV = exports.PJV;
 
-    PJV.getSpecMap = function (specName) {
+    PJV.getSpecMap = function (specName, isPrivate) {
 
         if (specName == "npm") {
             // https://docs.npmjs.com/cli/v9/configuring-npm/package-json
             return {
-                "name":         {"type": "string", required: true, format: PJV.packageFormat},
-                "version":      {"type": "string", required: true, format: PJV.versionFormat},
+                "name":         {"type": "string", required: !isPrivate, format: PJV.packageFormat},
+                "version":      {"type": "string", required: !isPrivate, format: PJV.versionFormat},
                 "description":  {"type": "string", warning: true},
                 "keywords":     {"type": "array", warning: true},
                 "homepage":     {"type": "string", recommended: true, format: PJV.urlFormat},
@@ -140,7 +140,7 @@
             return out;
         }
 
-        var map = PJV.getSpecMap(specName);
+        var map = PJV.getSpecMap(specName, parsed.private);
         if (specName === false) {
             out.critical = {"Invalid specification": specName};
             return out;
